src/curve.js

```diff
@@ -14,13 +14,17 @@
  * limitations under the License.
  */
 
+// Imported just for the side effect of getting the `types` it exports into
+// the type system during compile time.
+import './time';
```

Comment:
@cramforce so i had to import the module to get it to work (it wasnt used anywhere), is this necessary?